### ISSUE TITLE
Can't find QSql dependencies even though I don't use QSql

### ISSUE THREAD

**Issue by DavidLazarescu** [probonopd/linuxdeployqt#592]
I am getting the following method when trying to create an app image from my application.

![8f99234b-61c8-474b-a815-f926d9668e58](https://github.com/probonopd/linuxdeployqt/assets/69865187/be1f3e0c-d6b0-4e6f-99cd-ff71145985f6)

I only use the following Qt modules though: `find_package(Qt6 6.5 REQUIRED COMPONENTS Core Quick Widgets Network Gui QuickControls2 Test)` thus I suppose that this could be a bug.

I'd appreciate any help on this.

**Comment by probonopd** [comment]
Please check what might be drawing in sql by running, _prior to_ running linuxdeployqt, on your AppDir:

```
grep -i -e sql ./Your.AppDir
```

and the same for your qml directory.

**Comment by DavidLazarescu** [comment]
> Please check what might be drawing in sql by running, _prior to_ running linuxdeployqt, on your AppDir:
> 
> ```
> grep -i -e sql ./Your.AppDir
> ```
> 
> and the same for your qml directory.

Nothing in my src directory mentions sql

**Comment by DavidLazarescu** [comment]
The only thing that I was able to find is this in the CMakeCache.txt
![image](https://github.com/probonopd/linuxdeployqt/assets/69865187/b5fb8fc5-ab07-4477-b6e7-7d59a6ba3656)

**Comment by probonopd** [comment]
Oh, it's Qt6. I never even tested linuxdeployqt with that. Possibly you are right and it is related to that line, I don't know.

**Comment by y761823** [comment]
when I upgrade Qt to 6.6.1, I have the same problem (Qt 6.5.3 is ok)
seems that some qml things depend on SQL
log with `-verbose=3`:
```
log: Using ldd:
Log:  inspecting "/tmp/tmp.9grDFQLFtI/remote_operator/lib//libQt6QmlLocalStorage.so.6"
...
Log:  dylib.binaryPath "/home/ouyankai/Qt/6.6.1/gcc_64/lib/libQt6Sql.so.6"
...
```

I don't know how to solve or avoid it, I just installed the [mimersql](https://developer.mimer.com/products/downloads/mimer-sql-for-linux/) now

**Comment by DavidLazarescu** [comment]
Does it work when installing mimersql for you?

**Comment by y761823** [comment]
Yes. After I installed mimersql, I successfully released a app (test passed)

**Comment by danryu** [comment]
Also experienced this issue with Qt 6.10.0.
sqldrivers are being imported (and failing on mimersql) even though I don't use or reference sql anywhere in the project (QML C++).
